.github/workflows/coverage.yml

```
@@ -33,16 +33,21 @@ jobs:
         run: pnpm run build:libs
 
       - name: Run analysis coverage
-        run: pnpm --filter @financial-analysis/analysis run test:coverage
+        run: pnpm --filter @financial-analysis/analysis run test:coverage:ci
 
       - name: Upload coverage (Codecov)
+        if: ${{ secrets.CODECOV_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)